inspirehep/modules/theme/jinja2filters.py

```diff
@@ -613,18 +619,19 @@ def publication_info(record):
                         name="conference_only",
                         template="inspirehep_theme/format/record/Conference_info_macros.tpl",
                         ctx=ctx)
-                except TypeError:
-                    pass
             elif 'parent_recid' in pub_info and \
-                    'conference_recid' not in pub_info:
-                pid = PersistentIdentifier.get(
-                    'conferences', str(pub_info['parent_recid']))
-                parent_rec = es.get_source(
-                    index='records-hep',
-                    id=str(pid.object_uuid),
-                    doc_type='hep',
-                    ignore=404)
+                    'conference_recid' not in pub_info:                
                 try:
+                    pid = PersistentIdentifier.get(
+                        'conferences', str(pub_info['parent_recid']))
```

Comment:
Hmmm I am not quite sure about this. @jmartinm give us your lights here.
https://github.com/inspirehep/inspire-next/issues/541